File: Planner/src/app/acceuil/vol.ts
export class Vol {
    public Id:  Number;
    public isFlying: Boolean;
    public vol: string;
    public heureArr: string;
    public heureDepart: string;
    public departure: string;
    public arrival: string;
    public altitude: string;
    public idInfo: Number;

    constructor( isFlying: Boolean, vol: string, departure: string, arrival: string, altitude: string, idInfo: Number, heurearr:string, heureDepart:string){
this.isFlying = isFlying;
this.vol = vol;
this.departure = departure;
this.arrival = arrival;
this.altitude = altitude;
this.idInfo = idInfo;
this.heureArr= heurearr;
this.heureDepart = heureDepart;
    }

}

File: API/API/Models/Vol.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace API.Models
{
    public class Vol
    {

      
         [Key]
        public int Id { get; set; }

     
        public Boolean isFlying{ get; set; }

    
        public string vol { get; set; }

        public string heureDepart { get; set; }

        public string heureArr{ get; set; }

        public string departure { get; set; }

      
        public string arrival { get; set; }

        
        public string altitude { get; set; }

        [ForeignKey("InfoSup")]
        public int idInfo { get; set; }
        public InfoSup InfoSup{ get; set; }
    }
}
